frontend/services/api/sequences.ts

```diff
@@ -18,41 +18,48 @@
 import api from './';
 const HEAD = 'master/head';
 
+export const getSequence = (teamspace, modelId, sequenceId): Promise<any> => {
+	return api.get(`${teamspace}/${modelId}/sequences/${sequenceId}`);
+};
+
 /**
  * Get sequences list
  *
  * @param teamspace
  * @returns {*|promise}
  */
-export const getSequences = (teamspace, modelId, revision): Promise<any> => {
-	return api.get(`${teamspace}/${modelId}/revision/${revision || HEAD}/sequences`);
+export const getSequenceList = (teamspace, modelId, revision?): Promise<any> => {
+	if (revision) {
+		return api.get(`${teamspace}/${modelId}/sequences/?revId=${revision}`);
```

Comment:
ah that's why [this](https://github.com/3drepo/3drepo.io/pull/2537#issuecomment-827658045) isn't working! it needs to be rev_id